src/Command/ExtractElevationCommand.php

```
@@ -0,0 +1,87 @@
+      )
+    ;
+
+    $question = new Question('Please provide the path to the FIT file' . "\n > ");
+    $this->addInteractivityForOption('fitpath', self::INTERACTION_UNSET_ONLY, $question);
+
+    $question = new ChoiceQuestion('Please select json or csv for the output file' . "\n > ", ['json', 'csv']);
+    $this->addInteractivityForOption('output-format', self::INTERACTION_UNSET_ONLY, $question);
+  }
+
+// TODO: add testing
+  protected function execute(InputInterface $input, OutputInterface $output)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$output'. (Unused Code Rules)

(UnusedFormalParameter)